tests/test_answer_relevancy.py

```diff
@@ -24,6 +24,37 @@ def test_relevancy(self):
         score = metric.measure_zh(test_case)
         print(score)
 
+    @unittest.skip("need pass localhost id")
```

Comment:
Why skip the test yet write it?

Reply:
need a remote service, but we don't setup now